kong/plugins/log-serializers/basic.lua

```diff
@@ -12,11 +12,12 @@ function _M.serialize(ngx)
       consumer_id = ngx.ctx.authenticated_credential.consumer_id
     }
   end
-  
+
   return {
     request = {
-      uri = ngx.var.request_uri,
-      request_uri = ngx.var.scheme .. "://" .. ngx.var.host .. ":" .. ngx.var.server_port .. ngx.var.request_uri,
+      request_uri = ngx.var.request_uri,
+      uri = ngx.var.uri,
+      request_url = ngx.var.scheme.."://"..ngx.var.host..":"..ngx.var.server_port..ngx.var.request_uri,
```

Comment:
@shiprabehera thanks. Since this is a breaking change, we will need to point this change to the `next` branch (you can simply edit this PR), and will need to be rebased accordingly. We may also want to take some naming changes in https://github.com/Mashape/kong/commit/0010bce56279b61c98d824f17ca19e94a000c960 into account as well (not sure they're entirely relevant, but it's worth being aware of).